wallet-new/src/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -195,17 +196,43 @@ instance ToParamSchema Core.Address where
 instance ToParamSchema (V1 Core.Address) where
   toParamSchema _ = toParamSchema (Proxy @Core.Address)
 
+instance ( KnownSymbol sym
+         , HasSwagger sub
+         ) =>
+         HasSwagger (CustomQueryFlag sym flag :> sub) where
+    toSwagger _ =
+        let swgr       = toSwagger (Proxy @(QueryFlag sym :> sub))
+        in swgr & over (operationsOf swgr . parameters) (map toDescription)
+          where
+            toDescription :: Referenced Param -> Referenced Param
+            toDescription (Inline p@(_paramName -> pName)) =
+                case M.lookup pName customQueryFlagToDescription of
+                    Nothing -> Inline p
+                    Just d  -> Inline (p & description .~ Just d)
+            toDescription x = x
 
 --
 -- Descriptions
 --
 
+customQueryFlagToDescription :: Map T.Text T.Text
+customQueryFlagToDescription = M.fromList [
+    ("force_ntp_check", forceNtpCheckDescription)
+  ]
+
 requestParameterToDescription :: Map T.Text T.Text
 requestParameterToDescription = M.fromList [
     ("page", pageDescription)
   , ("per_page", perPageDescription (fromString $ show maxPerPageEntries) (fromString $ show defaultPerPageEntries))
   ]
 
+-- TODO: it would be nice to read ntp configuration directly here to fetcvh
+-- 30 seconds wait time instead of hardcoding it here.
+forceNtpCheckDescription :: T.Text
+forceNtpCheckDescription = [text|
+In some cases, Daedalus needs to force a new NTP check as a previous result gets cached. A typical use-case is after asking a user to fix its system clock. If this flag is set, request will block until NTP client responds or it will timout if NTP client is not available within **30** seconds.
```

Comment:
I wouldn't mention Daedalus here but merely `API Clients`

Reply:
Heh I wondered how you will react on this "more releaxed" description style :D Guess I know now

Reply:
Haha, in theory, _Daedalus_-specific stuffs are only under the `Internal` API. There's no reason to mention Daedalus anywhere else :p